src/libs/kata-types/src/config/agent.rs

```diff
@@ -80,6 +80,24 @@ pub struct Agent {
     pub container_pipe_size: u32,
 }
 
+impl std::default::Default for Agent {
+    fn default() -> Self {
+        Self {
+            debug: true,
+            enable_tracing: false,
+            server_port: DEFAULT_AGENT_VSOCK_PORT,
+            log_port: DEFAULT_AGENT_LOG_PORT,
+            dial_timeout_ms: DEFAULT_AGENT_DIAL_TIMEOUT_MS,
+            reconnect_timeout_ms: 3_000,
+            request_timeout_ms: 30_000,
+            health_check_request_timeout_ms: 90_000,
+            kernel_modules: Default::default(),
+            container_pipe_size: 0,
+            debug_console_enabled: false,
```

Comment:
nit: this field is out of order from how they are defined above. It will improve readability to list it under `enable_tracing` before `server_port`.

Reply:
Done.